docs/master/api-reference/directives.md

```diff
@@ -1074,6 +1074,39 @@ directive @in(
 ) on ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION
 ```
 
+## @include
+
+This directive is part of the [GraphQL spec](https://graphql.github.io/graphql-spec/June2018/#sec--include) 
```

Comment:
Let's not have this directive listed here and keep this reference limited to schema directives. We should find another place for your writeup though, it is quite good.

Reply:
@spawnia That makes sense. Do you have any ideas on where we should put it? I'd be happy to add that to my PR.

Reply:
`Digging Deeper/Client Directives` seems like a good location. We might add a guide on how to add custom client directives as well.

Reply:
@spawnia Perfect. I'll try to have that added sometime in the next day or so. That would be really cool to have some docs on client directives. I actually did not realize that was even possible until I came across these.

Reply:
Done it